### PR TITLE
OPENEUROPA-2229: Fix field translatability

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Fix field translatability.

### CODE REVIEW

File: modules/oe_content_persistent/tests/src/Functional/PersistentUrlControllerTest.php [hunk 1]
```diff
@@ -38,6 +38,9 @@ protected function setUp() {
 
     $node_type = NodeType::create(['type' => 'page']);
     $node_type->save();
+
+    $user = $this->drupalCreateUser(['access content']);
```
Comment (upchuk) [inline modules/oe_content_persistent/tests/src/Functional/PersistentUrlControllerTest.php]: Why is this needed actually?

File: modules/oe_content_policy/oe_content_policy.post_update.php [hunk 2]
```diff
@@ -53,3 +53,24 @@ function oe_content_policy_post_update_00002_field_labels(array &$sandbox): void
     $field_config->save();
   }
 }
+
+/**
+ * Make the timeline field translatable and RDF fields not translatable.
+ */
+function oe_content_policy_post_update_00003_update_translatable_fields(array &$sandbox): void {
```
Comment (upchuk) [inline modules/oe_content_policy/oe_content_policy.post_update.php]: We cannot change existing sites' translatability. So we need to move this update path to EWCMS.

File: modules/oe_content_publication/oe_content_publication.post_update.php [hunk 3]
```diff
@@ -22,3 +22,19 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
     $field_config->save();
   }
 }
+
+/**
+ * Make RDF fields not translatable.
+ */
+function oe_content_publication_post_update_00002_not_translatable_rdf_field(array &$sandbox): void {
```
Comment (upchuk) [inline modules/oe_content_publication/oe_content_publication.post_update.php]: We cannot change existing sites' translatability. So we need to move this update path to EWCMS.

File: modules/oe_content_persistent/tests/src/Functional/PersistentUrlControllerTest.php (hunk 1 above)
Comment (voidtek) [reply modules/oe_content_persistent/tests/src/Functional/PersistentUrlControllerTest.php]: related to https://github.com/openeuropa/oe_content/pull/98

File: modules/oe_content_policy/oe_content_policy.post_update.php (hunk 2 above)
Comment (voidtek) [reply modules/oe_content_policy/oe_content_policy.post_update.php]: done here https://github.com/ec-europa/ewcms/pull/197

File: modules/oe_content_publication/oe_content_publication.post_update.php (hunk 3 above)
Comment (voidtek) [reply modules/oe_content_publication/oe_content_publication.post_update.php]: done here https://github.com/ec-europa/ewcms/pull/197
